azure-pipelines.yml

```
@@ -259,6 +299,34 @@ jobs:
     inputs:
       filePath: build_scripts\windows\scripts\test_msi_installation.ps1
 
+- job: TestZipPackage
```

The name `TestZipPackage` doesn't tell whether we are testing the installation or contents in the package (unit tests, scenario tests, etc).

I would suggest we align with `TestMsiInstallation`:

```suggestion
- job: TestZipInstallation
```